src/neighbors.cc

```diff
@@ -20,6 +20,8 @@
 #include "CachePeers.h"
 #include "comm/Connection.h"
 #include "comm/ConnOpener.h"
+#include "compat/netdb.h"
+#include "compat/socket.h"
```

Comment:
API provided by this include is not used in this .cc file.
```suggestion
```

Reply:
removed